thesis_text/thesis_text.md

```diff
@@ -0,0 +1,551 @@
+# Truffle based .NET IL interpreter and compiler: run C# on Java Virtual Machine
+
+# Abstract
+
+Interpreted and just-in-time (JIT) compiled languages are becoming more and more prevalent, with JavaScript and Python ranking in TOP 5 most popular programming languages in most surveys and statistics. In the academic and research settings, the ability to quickly prototype programming languages and test their properties also works in favor of interpreted languages, as implementing an interpreter is considerably easier than implementing a compiler. However, simple interpreters suffer from a substantial tradeoff of performance. Traditionally, if an interpreted language is to become effective and run fast, it requires creating a custom infrastructure of JIT compilers and optimizations just for that single specific language.
```

Comment:
Není mi jasné, co se tím chce říct: "Interpreted and just-in-time (JIT) compiled languages are becoming more and more prevalent" -- vs. co? AOT compiled languages? Dále: "works in favor of interpreted languages" -- takže nás zajimají interpreter languages? Klasický nitpick koholiv z prostředí PL research by byl, že neexistují interpretované ani kompilované jazyky, ale jejich implementace.

Obecně tahle kapitola míří dobrým směrem: zdůvodnit, proč mělo smysl to celé dělat -- že chceme zjistit, jestli to půjde s Trufflem rychleji a/nebo to bude v něčem lepší než tradiční přístup.

Reply:
Tohle je vykopírovaný abstract ze zadání práce který už je dlouho zapsaný v SISu, s ním už nic neudělám :)

Reply:
Ah, ja jsem si rikal. Njn